openssl-dynamic/src/main/c/ssl.c

```diff
@@ -392,6 +397,23 @@ static int ssl_ui_writer(UI *ui, UI_STRING *uis)
 }
 #endif // OPENSSL_NO_ENGINE
 
+TCN_IMPLEMENT_CALL(void, SSL, bioSetFd)(TCN_STDARGS, jlong ssl, jint fd) {
+    SSL *ssl_ = J2P(ssl, SSL *);
+
+    if (ssl_ == NULL) {
+      tcn_ThrowException(e, "ssl pointer is null!");
+    }
```

Comment:
Use the following macro. This Will also ensure we return correctly in case of failure which you didn't do here and so would cause a segfault later 
```suggestion
    TCN_CHECK_NULL(ssl_, ssl, /* void */);
```